### ISSUE TITLE
Problem with cyrillic charset

### ISSUE THREAD

**Issue by cvtk** [yvesh/active-windows#9]
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: "╨Ф╨╢╨╕╨╜╤Б╤Л 511 Slim Fit Levi's┬о 6407635 ╨║╤Г╨┐╨╕╤В╤М ╨╖╨░ 3┬а544┬атВ╜ ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В-╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ Wildberries - Google Chrome",
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: "╨Ф╨╢╨╕╨╜╤Б╤Л 511 Slim Fit Levi's┬о 6407635 ╨║╤Г╨┐╨╕╤В╤М ╨╖╨░ 3┬а544┬атВ╜ ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В-╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ Wildberries - Google Chrome",
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: '╨Ъ╤Г╨┐╨╕╤В╤М ╨╝╤Г╨╢╤Б╨║╨╕╨╡ ╨┤╨╢╨╕╨╜╤Б╤Л ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В ╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ WildBerries.ru - Google Chrome',
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: '╨Ъ╤Г╨┐╨╕╤В╤М ╨╝╤Г╨╢╤Б╨║╨╕╨╡ ╨┤╨╢╨╕╨╜╤Б╤Л ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В ╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ WildBerries.ru - Google Chrome',
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: '╨Ъ╤Г╨┐╨╕╤В╤М ╨╝╤Г╨╢╤Б╨║╨╕╨╡ ╨┤╨╢╨╕╨╜╤Б╤Л ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В ╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ WildBerries.ru - Google Chrome',
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}
{
  os: 'windows',
  windowClass: 'chrome.exe',
  windowName: '╨Ъ╤Г╨┐╨╕╤В╤М ╨╝╤Г╨╢╤Б╨║╨╕╨╡ ╨┤╨╢╨╕╨╜╤Б╤Л ╨▓ ╨╕╨╜╤В╨╡╤А╨╜╨╡╤В ╨╝╨░╨│╨░╨╖╨╕╨╜╨╡ WildBerries.ru - Google Chrome',
  windowDesktop: '0',
  windowType: '0',
  windowPid: '9608',
  idleTime: '0'
}

**Comment by yvesh** [comment]
Yes, Windows is sometimes a mess with it's charsets. Not even sure which one it's using for cyrillic / russian (not UTF-8 it seems). Have it on my todo-list - PRs welcome though!

**Comment by cvtk** [comment]
@yvesh Hi!
I would like to take part, but there is only one thing better than my C++ code - my poor english

**Comment by nodgear** [comment]
Hey! i'm currently using FFI so i don't know if it's going to be of any help, but i am currently getting the correct window names this way:
```js
 const buf = new Buffer.alloc(1024)
  const ret = win32.GetWindowTextW(hwnd, buf, 1024)
  const name = buf.toString('ucs2').replace(/\0+$/, '')
  ```
  Where win32 is the user32 dll via ffi-napi library.
  
  @Edit: this does include emojis. Windows seems to not have a limit for window title size so it may be a problem sometimes.
